### PR TITLE
models: bucket tags

### PR DESCRIPTION
**Pull request by lnielsen**

### CODE REVIEW

**Comment by lnielsen** [comment]
Tags as discussed IRL

**Comment by egabancho** [comment]
LGTM, @krzysztof?

File: examples/app.py [hunk 1]
```diff
@@ -133,22 +142,22 @@ def files():
     # Bucket 1
     b1 = Bucket.create(loc)
     for f in ['README.rst', 'LICENSE']:
-        with open(join(srcroot, f), 'r') as fp:
-            Object.create(b1, f, stream=fp)
+        with open(join(srcroot, f), 'rb') as fp:
```
Comment (krzysztof) [inline examples/app.py]: Not a real issue but for testing purposes I find ``BytesIO`` object to be a cleaner solution, see for example: https://github.com/krzysztof/invenio-files-rest/blob/permissions/tests/test_permissions.py#L40

**Comment by krzysztof** [comment]
Well, I guess it LGTM as well then ;)

**Pull request merged**